editor/code/views/info/Info.tsx

```diff
@@ -1,15 +1,22 @@
-import React, { useEffect, useState } from "react";
+import { useEffect, useState, lazy, Suspense } from "react";
 import { GoalAnswer, GoalRequest, PpString } from "../../lib/types";
 
-// import "./media/info.css";
-
 // Main panel components
 import { FileInfo } from "./FileInfo";
 import { Goals } from "./Goals";
 import { Messages } from "./Messages";
 import { ErrorBrowser } from "./ErrorBrowser";
 import { Program } from "./Program";
 
+import "./media/info.css";
+
+// Dynamic import because the project uses CommonJS and the module is an ECMAScript module
+// Top level await is supported with other `module` options in tsconfig.json
+const VSCodeDivider = lazy(async () => {
+  const { VSCodeDivider } = await import("@vscode/webview-ui-toolkit/react");
+  return { default: VSCodeDivider };
+});
```

Comment:
Don't know if there's a better way but it kept complaining about CommonJS and ECMAScript modules and this is the only way I got it working. Otherwise there might be other options to change in `tsconfig.json` but I'm not very familiar with the consequences.

Reply:
I got it working in `views/perf` just using:
```typescript
import {
  VSCodeDataGrid,
  VSCodeDataGridRow,
  VSCodeDataGridCell,
} from "@vscode/webview-ui-toolkit/react";
```
Where did you get the complaints?